### DISCUSSION TITLE
Code freeze starting today, in preparation for the 1.19 release

### DISCUSSION THREAD

**Discussion by jsilvela** [cloudnative-pg/cloudnative-pg#1388] [Release updates]
The next release of CloudNativePG is scheduled in a week, for 2023/02/09
In preparation, we have entered code freeze. In practice this means we want
to focus our resources in finishing the tickets that are meant to land in 1.19,
and holding other tickets back until after release.

We are tracking this with issue https://github.com/cloudnative-pg/cloudnative-pg/issues/1387

**Comment by gbartolini** [comment]
Due to the discovery of a last minute bug (#1422), we are postponing the release by a few days to 14 Feb, 2023.
